### PR TITLE
Prevent players to throw frag/smoke grenade on build time

### PR DESCRIPTION
**Pull request by DiamondPlane**

### CODE REVIEW

File: mods/pvp/grenades/grenades.lua [hunk 1]
```diff
@@ -16,6 +16,14 @@ local fragdef = {
...
 			return
 		end
+                if ctf_match.is_in_build_time() then
+		local player = minetest.get_player_by_name(name)
+                if player then
+			player:get_inventory():add_item("main", "grenades:frag")
+                        return
+		        minetest.chat_send_player(name, "Match hasn't started yet!")
+                end
+        end
```
Comment (Panquesito7) [inline mods/pvp/grenades/grenades.lua:26]: Wrong indentation.

File: mods/pvp/grenades/grenades.lua [hunk 2]
```diff
@@ -94,6 +102,13 @@ grenades.register_grenade("grenades:smoke", {
 	on_explode = function(pos, pname)
 		local player = minetest.get_player_by_name(pname)
 		if not player or not pos then return end
+                if ctf_match.is_in_build_time() then
+                if player then
+			player:get_inventory():add_item("main", "grenades:smoke")
+                        return
+		        minetest.chat_send_player(pname, "Match hasn't started yet!")
+                end
+        end
```
Comment (Panquesito7) [inline mods/pvp/grenades/grenades.lua:111]: Same here as above.

**Review by Panquesito7 — APPROVED** [review]
👍 apart from the above (tested).

**Comment by Toby1710** [comment]
Hmm... Can you suggest changes me to make the indentation correct, please?

**Review by TSafa-23 — CHANGES_REQUESTED** [review]
Be sure to fix redundancies/indentation otherwise it likely won't be merged. Also, does this prevent grenades from being thrown or only giving them back if they were thrown before the build time is up?

**Comment by Toby1710** [comment]
https://github.com/MT-CTF/capturetheflag/pull/804#discussion_r579849054 @TSafa-23  I tested with this, it gave me error. But on frag it didn't

**Comment by TSafa-23** [comment]
Was more of an example because you asked to see one btw, just so you know how it roughly should look

**Comment by Toby1710** [comment]
> Was more of an example because you asked to see one btw, just so you know how it roughly should look

Should I leave the code that I did?

**Comment by DiamondPlane** [comment]
I have a problem with this PR, I'll create a new one.

**Pull request closed without merging**